src/app/children/child-details/child-details.component.ts

```diff
@@ -109,6 +109,8 @@ export class ChildDetailsComponent implements OnInit {
       dropoutDate:    [{value: this.child.dropoutDate,    disabled: !this.editing}],
       dropoutType:    [{value: this.child.dropoutType,    disabled: !this.editing}],
       dropoutRemarks: [{value: this.child.dropoutRemarks, disabled: !this.editing}],
+
+      hasPhoto: [this.child.hasPhoto]
```

Comment:
should we maybe save the filename instead? That'd give us more flexibility

Reply:
Done.